.github/workflows/ci.yml

```diff
@@ -2,9 +2,12 @@ name: CI
 
 on:
   push:
-    branches: [ main ]
+    branches:
+      - 'main'
+      - 'test-feature-BL-15047-doctrine-cache'
```

Comment:
I don't think you need this line, as the `pull_request` key below handles PRs that are being merged into main (https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpull_requestpull_request_targetbranchesbranches-ignore)

Reply:
oh yeah this was for testing, will remove now